src/store/actions/posting-project-user/project/project.js

```diff
@@ -129,54 +129,70 @@ export const _createProject_ppu = (
   deadline,
   benefits,
   requirements,
-  imageFIle,
+  imageFile,
   questions,
   category = "General"
 ) => {
+  //Handle data error
+  if (orgId == "") orgId = JSON.parse(localStorage.getItem("userData")).userId;
+  if (!deadline) deadline = "";
+
+  //For debugging purpose
+  const project = {
+    orgId: orgId,
+    projectId: projectId,
+    name: name,
+    description: description,
+    location: location,
+    deadline: deadline,
+    benefits: benefits,
+    requirements: requirements,
+    imageFIle: imageFile,
+    questions: questions,
+    category: category,
+  };
+  console.log("Project to push: ", project);
+
+  //Fire store management
   return async (dispatch, getState) => {
-    const projectImageUrl = await _imageUploadHandler(imageFIle);
+    const projectImageUrl = await _imageUploadHandler(imageFile);
     console.log("projectImageUrl", projectImageUrl);
-    const projectRef = firebase_db
-      .collection("public-projects")
-      .doc(`${projectId}`);
-    const organizationRef = firebase_db
-      .collection("organization")
-      .doc(`${orgId}`);
+    const projectRef = firebase_db.collection("public-projects").doc(projectId);
+    const organizationRef = firebase_db.collection("organization").doc(orgId);
```

Comment:
Wrong syntax. 
```suggestion
    const projectRef = firebase_db.collection("public-projects").doc(`${projectId}`);
    const organizationRef = firebase_db.collection("organization").doc(`${orgId});
```

Reply:
![image](https://user-images.githubusercontent.com/66483590/96073051-ce793080-0ecf-11eb-93fe-c47c5b6221ce.png)

Why with this one it's correct